src/controllers/drivers.cr

```diff
@@ -52,19 +49,19 @@ module PlaceOS::Core::Api
       )
 
       execute_output = io.to_s
+      temporary_driver_path = result[:executable]
 
-      # Remove the driver if it was compiled for the lifetime of the query
-      File.delete(temporary_driver_path) if temporary_driver_path && File.exists?(temporary_driver_path)
+      # Remove the driver as it was compiled for the lifetime of the query
+      File.delete(temporary_driver_path) if File.exists?(temporary_driver_path)
 
-      if result.exit_status != 0
-        logger.tag_error("failed to execute", **(meta.merge({output: execute_output}))
-        )
+      if result.exit_code != 0
+        logger.tag_error("failed to execute", **(meta.merge({output: execute_output})))
         render :internal_server_error, json: {
-          exit_status: result.exit_status,
+          exit_status: result.exit_code,
           output:      execute_output,
           driver:      driver,
           version:     commit,
-          executable:  executable_path,
+          executable:  executable_path, # TODO: Remove field
```

Comment:
Yep, agree we should remove this. If it breaks rest-api, lets fix it there once this is up